src/Store/MultiTableStore.php

```diff
@@ -96,7 +97,7 @@ public function has(string $aggregate, string $id): bool
 
     /**
      * @param class-string<AggregateRoot> $aggregate
-     * @param AggregateChanged[]          $events
+     * @param array<AggregateChanged>     $events
```

Comment:
```suggestion
     * @param list<AggregateChanged> $events
```

Reply:
cant be done unless using `array_values` after `array_merge`. Seems not feasible for this